alpakka/src/test/scala/org/scanamo/ScanamoAlpakkaSpec.scala

```diff
@@ -641,4 +641,97 @@ class ScanamoAlpakkaSpec extends FunSpecLike with BeforeAndAfterAll with Matcher
       }
     }
   }
+
+  it("transact table write (update) items") {
```

Comment:
There's a fair but of duplication between the different specs here, not sure if we see some value in reducing that. For now I've just followed the existing style which seems ok.

Reply:
Agreed. There is a way around that using some form of `ZManaged` or similar and feeding tables with a random sample of data, then all these tests will be reduced to "laws". One day 😩